core/templates/domain/utilities/language-util.service.ts

```diff
@@ -182,6 +186,11 @@ export class LanguageUtilService {
     return language ? language.description : null;
   }
 
+  languageCodeIsMachineTranslatable(languageCode: string): boolean {
```

Comment:
I believe it's more conventional to have "is" be the leading prefix.

```suggestion
  isLanguageCodeMachineTranslatable(languageCode: string): boolean {
```